src/Task/Task.php

```
@@ -77,16 +77,20 @@ public function __construct(string $name, ?callable $callback = null)
     }
 
     /**
-     * @param callable():void $callback
+     * @param any $callback
      */
-    public function setCallback(callable $callback): void
+    public function setCallback($callback): void
```

PHPStan (2.1.15)

Parameter $callback of method Deployer\Task\Task::setCallback() has invalid type Deployer\Task\any.


(class.notFound)